dashboard/app/controllers/levels_controller.rb

```diff
@@ -43,7 +43,7 @@ def edit_blocks
     @level = Level.find(params[:level_id])
     authorize! :edit, @level
     type = params[:type]
-    blocks_xml = @level.properties[type].presence || @level[type]
+    blocks_xml = @level.properties[type].presence || @level[type] || '<xml></xml>'
```

Comment:
it seems like herewe use <xml></xml> and sometimes (like above in StudioApp) we use '', is there a reason for that? Does something filter out .blank?/.empty? properties?